agr_java_core/src/main/java/org/alliancegenome/core/helpers/DiseaseAnnotationHelper.java

```diff
@@ -18,7 +18,7 @@ public static String getEntityName(BiologicalEntity entity) {
 			return allele.getAlleleSymbol().getFormatText();
 		}
 		if (entity instanceof org.alliancegenome.curation_api.model.entities.AffectedGenomicModel model) {
-			return model.getAgmFullName().getDisplayText();
+			return model.getAgmFullName() != null ? model.getAgmFullName().getDisplayText() : "no name provided for this AGM";
```

Comment:
I know @chris-grove said this isn't always the case, however I looked yesterday in the DB and right now in production ALL AGM's have names. So not sure why we were hitting null. I think we need to figure out why before moving ahead with this change?